helm/fcp-fd-file-receiver/templates/azure-identity.yaml

```diff
@@ -0,0 +1,3 @@
+{{- include "ffc-helm-library.azure-identity" (list . "fcp-fd-file-receiver.azure-identity") -}}
```

Comment:
include the IF clause in this file as here:
https://github.com/DEFRA/fcp-fd-data/blob/main/helm/fcp-fd-data/templates/azure-identity.yaml

Dont forget to add the new value into the `values.yaml`